lib/ui/page/swap.dart

```diff
@@ -210,6 +210,23 @@ class _Body extends HookWidget {
                     ),
                   ),
                 )),
+            Expanded(
+                child: SelectableText.rich(
```

Comment:
missed set `enableInteractiveSelection` to false ?

Reply:
web content can not be selected seems kinda weird